mobileApp/jobs/buildWhiteLabelAndroidApps.groovy

```diff
@@ -8,10 +8,16 @@ Example secret YAML file used by this script
 jobConfig:
     jobName: build-demo-android-app
     public: true
+    release: false
```

Comment:
Ah. Well, yes that would be helpful. An open source person woudl more-easily understand what's going on. To a certain extent I'd rather see the comments in the open source repo rather than the secrets repo for that reason. And it would likely apply to not-just open source folks; it would apply to edX folks who don't have access to the secrets.

You don't have to solve that particular pattern here, but perhaps you can add the comment to this line?